tests/test_entities.py

```diff
@@ -166,6 +166,7 @@ def test_init_succeeds(self):
                 entities.SmartClassParameters,
                 entities.SmartProxy,
                 entities.SmartVariable,
+                # entities.Snapshot,  # see below
```

Comment:
Is there any reason for this? Or can we just remove it?

Reply:
It has been seen before. I think it should make the list of tested entities complete with the ones needing special treatement commented out.
https://github.com/SatelliteQE/nailgun/pull/634/files#diff-ee816b7bbef31c47098eb9d90bf6da01R112